vercel/src/lib/AsciiPreview.svelte

```diff
@@ -0,0 +1,86 @@
+<script lang="ts">
+  import Chip from "./Chip.svelte";
+  import type { LanguageColors } from "../types";
+
+  export let name: string;
+  export let ansi: string[];
+  export let hex: string[] | null = null;
+  export let chip: string;
+  export let ascii: string = "";
+
+  let dark = true;
+  let trueColor = hex != null;
+
+  $: html = ascii
+    .split("\n")
+    .map((line) => {
+      // TODO Clean up, this is hard to read
+      let spanCount = 0;
+      const htmlLine = line.replace(/\{(\d+)\}/g, (_match, index) => {
+        const i = Number.parseInt(index, 10);
+        const spanText = trueColor
+          ? `<span style="color: ${mapToDefaultTerminalFgColor(hex[i], dark)}">`
```

Comment:
Should we remap the true color? The onefetch binary doesn't remap true colors, and it might be useful to soo how the logo will render if it's white-on-white.

Reply:
You're right, I thought we did, my bad 🤦 
The whole point of the ascii preview is to be iso with the binary.